.circleci/config.yml

```diff
@@ -1,8 +1,8 @@
 version: 2.1
 
 orbs:
-  codacy: codacy/base@5.2.0
-  codacy_plugins_test: codacy/plugins-test@0.15.4
+  codacy: codacy/base@10.2.0
+  codacy_plugins_test: codacy/plugins-test@2.0.0
```

Comment:
```suggestion
  codacy_plugins_test: codacy/plugins-test@1.1.1
```

Reply:
The latest version [seems to be 2.0.0](https://circleci.com/developer/orbs/orb/codacy/plugins-test), but apparently [that one didn't work](https://circleci.com/developer/orbs/orb/codacy/plugins-test). :thinking:

Let's use the previous version then. :ok_hand:

Reply:
Yup, I tried the latest one but it would not work, so I reverted to previous one 👍